eplaunch/interface/filenamemenus.py

```diff
@@ -36,8 +36,8 @@ def get_file_list(self):
             if menu_item.GetId() == self.end_separator_id:
                 mode = False
             if mode:
-                list_of_menu_item_labels.append(menu_item.GetLabel())
-                self.file_name_to_id[menu_item.GetLabel()] = menu_item.GetId()
+                list_of_menu_item_labels.append(menu_item.GetItemLabel())
+                self.file_name_to_id[menu_item.GetItemLabel()] = menu_item.GetId()
```

Comment:
My IDE was complaining about `GetLabel` -- need to test this on all the platforms prior to merging.

Reply:
Works on Windows 10